metadata/containers.go

```diff
@@ -144,7 +151,9 @@ func (s *containerStore) Create(ctx context.Context, container containers.Contai
 		if err := writeContainer(cbkt, &container); err != nil {
 			return fmt.Errorf("failed to write container %q: %w", container.ID, err)
 		}
-
+		span.SetAttributes(
+			tracing.Attribute("container.CreatedAt", container.CreatedAt.Format(time.RFC3339)),
```

Comment:
createdAt or CreatedAt?

Reply:
will change this to use smaller case like the rest.